### ISSUE TITLE
Exception with 2023.2 EAP 232.8453.116

### ISSUE THREAD

**Issue by MCMicS** [MCMicS/jenkins-control-plugin#488]
Investigate if Jetbrains issue or of plugin

```
IntelliJ IDEA 2023.2 EAP (Ultimate Edition)
Build #IU-232.8453.116, built on June 29, 2023
Runtime version: 17.0.7+7-b1000.2 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 11.0
Jenkins Plugin 0.13.18-eap1-2022.2
GC: G1 Young Generation, G1 Old Generation
Memory: 3096M
Cores: 24
Registry:
    debugger.new.tool.window.layout=true
    ide.plugins.snapshot.on.unload.fail=true
    ide.experimental.ui=true
    ide.images.show.chessboard=true

Non-Bundled Plugins:
    com.krylysov.nsisplugin (0.2.0)
    CMD Support (1.0.5)
    PMDPlugin (1.8.27)
    com.jetbrains.edu (2023.6-2023.2-72)
    org.jetbrains.plugins.localization (232.8453.111)
    com.intellij.kubernetes (232.8453.111)
    Jenkins Control Plugin (0.13.18-eap1-2022.2)
    org.sonarlint.idea (8.4.0.73538)
    com.intellij.plugins.html.instantEditing (232.8453.111)
    com.jetbrains.php (232.8453.129)
    com.jetbrains.php.wordPress (232.8453.129)
    Pythonid (232.8453.116)
    CheckStyle-IDEA (5.81.0)
    MavenRunHelper (4.23.222.2964.0)
    com.google.gct.core (23.6.3-api-version-222)
    ru.adelf.idea.dotenv (2023.2)
    com.google.idea.bazel.ijwb (2023.07.06.0.1-api-version-232)

Kotlin: 232-1.8.21-release-380-IJ8453.116
```

```
java.lang.IllegalArgumentException: miter limit < 1
	at java.desktop/java.awt.BasicStroke.<init>(BasicStroke.java:201)
	at java.desktop/java.awt.BasicStroke.<init>(BasicStroke.java:249)
	at com.github.weisj.jsvg.M.a(SourceFile:70)
	at com.github.weisj.jsvg.cR.a(SourceFile:237)
	at com.github.weisj.jsvg.be.a(SourceFile:146)
	at com.github.weisj.jsvg.br.a(SourceFile:1094)
	at com.github.weisj.jsvg.br.a(SourceFile:1094)
	at com.github.weisj.jsvg.br.a(SourceFile:1094)
	at com.github.weisj.jsvg.bp.a(SourceFile:141)
	at com.github.weisj.jsvg.nodes.SVG.renderWithSize(SourceFile:135)
	at com.intellij.ui.svg.JsvgKt.renderSvgWithSize(jsvg.kt:46)
	at com.intellij.ui.svg.JsvgKt.renderSvg(jsvg.kt:19)
	at com.intellij.ui.svg.JsvgKt.renderSvg$default(jsvg.kt:10)
	at com.intellij.ui.svg.SvgKt.renderImage(svg.kt:309)
	at com.intellij.ui.svg.SvgKt.renderAndCache-rdQnj48(svg.kt:261)
	at com.intellij.ui.svg.SvgKt.access$renderAndCache-rdQnj48(svg.kt:1)
	at com.intellij.ui.svg.SvgKt.loadSvgAndCacheIfApplicable-F6nGey4(svg.kt:590)
	at com.intellij.ui.icons.ImageCacheKt.doLoadByDescriptor(imageCache.kt:290)
	at com.intellij.ui.icons.ImageCacheKt.loadByDescriptorWithoutCache(imageCache.kt:177)
	at com.intellij.ui.icons.ImageCacheKt.loadImage(imageCache.kt:139)
	at com.intellij.ui.icons.ImageDataByPathResourceLoader.loadImage(ImageDataByUrlLoader.kt:65)
	at com.intellij.ui.icons.CachedImageIcon.loadImage$intellij_platform_util_ui(CachedImageIcon.kt:313)
	at com.intellij.ui.icons.ScaledIconCache.getOrScaleIcon(ScaledIconCache.kt:43)
	at com.intellij.ui.icons.CachedImageIcon.resolveActualIcon(CachedImageIcon.kt:158)
	at com.intellij.ui.icons.CachedImageIcon.getIconWidth(CachedImageIcon.kt:119)
	at com.intellij.ui.RowIcon.updateSize(RowIcon.java:170)
	at com.intellij.ui.RowIcon.<init>(RowIcon.java:44)
	at org.codinjutsu.tools.jenkins.view.JenkinsTreeRenderer$CompositeIcon.<init>(JenkinsTreeRenderer.java:216)
	at org.codinjutsu.tools.jenkins.view.JenkinsTreeRenderer.render(JenkinsTreeRenderer.java:195)
	at org.codinjutsu.tools.jenkins.view.JenkinsTreeRenderer$1.visit(JenkinsTreeRenderer.java:71)
	at org.codinjutsu.tools.jenkins.view.JenkinsTreeNode$JobNode.render(JenkinsTreeNode.java:112)
	at org.codinjutsu.tools.jenkins.view.JenkinsTreeRenderer.render(JenkinsTreeRenderer.java:58)
	at org.codinjutsu.tools.jenkins.view.JenkinsTreeRenderer.lambda$customizeCellRenderer$0(JenkinsTreeRenderer.java:54)
	at java.base/java.util.Optional.ifPresent(Optional.java:178)
	at org.codinjutsu.tools.jenkins.view.JenkinsTreeRenderer.customizeCellRenderer(JenkinsTreeRenderer.java:54)
	at com.intellij.ui.ColoredTreeCellRenderer.rendererComponentInner(ColoredTreeCellRenderer.java:126)
	at com.intellij.ui.ColoredTreeCellRenderer.getTreeCellRendererComponent(ColoredTreeCellRenderer.java:58)
	at com.intellij.ui.tree.ui.DefaultTreeUI.getRenderer(DefaultTreeUI.java:189)
	at com.intellij.ui.tree.ui.DefaultTreeUI.paint(DefaultTreeUI.java:331)
	at java.desktop/javax.swing.plaf.ComponentUI.update(ComponentUI.java:161)
	at java.desktop/javax.swing.JComponent.paintComponent(JComponent.java:845)
	at com.intellij.ui.treeStructure.Tree.paintComponent(Tree.java:346)
	at com.intellij.ui.treeStructure.SimpleTree.paintComponent(SimpleTree.java:222)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1115)
	at com.intellij.ui.treeStructure.Tree.paint(Tree.java:263)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JViewport.paint(JViewport.java:736)
	at com.intellij.ui.components.JBViewport.paint(JBViewport.java:237)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JLayeredPane.paint(JLayeredPane.java:586)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JLayeredPane.paint(JLayeredPane.java:586)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1124)
	at java.desktop/javax.swing.JLayeredPane.paint(JLayeredPane.java:586)
	at java.desktop/javax.swing.JComponent.paintChildren(JComponent.java:955)
	at java.desktop/javax.swing.JComponent.paintToOffscreen(JComponent.java:5319)
	at java.desktop/javax.swing.RepaintManager$PaintManager.paintDoubleBufferedFPScales(RepaintManager.java:1732)
	at java.desktop/javax.swing.RepaintManager$PaintManager.paintDoubleBuffered(RepaintManager.java:1641)
	at java.desktop/javax.swing.RepaintManager$PaintManager.paint(RepaintManager.java:1580)
	at java.desktop/javax.swing.RepaintManager.paint(RepaintManager.java:1347)
	at java.desktop/javax.swing.JComponent.paint(JComponent.java:1101)
	at java.desktop/java.awt.GraphicsCallback$PaintCallback.run(GraphicsCallback.java:39)
	at java.desktop/sun.awt.SunGraphicsCallback.runOneComponent(SunGraphicsCallback.java:75)
	at java.desktop/sun.awt.SunGraphicsCallback.runComponents(SunGraphicsCallback.java:112)
	at java.desktop/java.awt.Container.paint(Container.java:2005)
	at java.desktop/java.awt.Window.paint(Window.java:3987)
	at com.intellij.openapi.wm.impl.IdeFrameImpl.paint(IdeFrameImpl.kt:106)
	at java.desktop/javax.swing.RepaintManager$4.run(RepaintManager.java:893)
	at java.desktop/javax.swing.RepaintManager$4.run(RepaintManager.java:865)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/javax.swing.RepaintManager.paintDirtyRegions(RepaintManager.java:865)
	at java.desktop/javax.swing.RepaintManager.paintDirtyRegions(RepaintManager.java:838)
	at java.desktop/javax.swing.RepaintManager.prePaintDirtyRegions(RepaintManager.java:787)
	at java.desktop/javax.swing.RepaintManager$ProcessingRunnable.run(RepaintManager.java:1909)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:789)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:740)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:734)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:759)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:685)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$10(IdeEventQueue.kt:589)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWithoutImplicitRead(ApplicationImpl.java:1485)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:589)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:992)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:992)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:400)
	at com.intellij.openapi.actionSystem.impl.Utils.lambda$expandActionGroupImpl$3(Utils.java:260)
	at com.intellij.openapi.actionSystem.impl.Utils.runLoopAndWaitForFuture(Utils.java:869)
	at com.intellij.openapi.actionSystem.impl.Utils.expandActionGroupImpl(Utils.java:258)
	at com.intellij.openapi.actionSystem.impl.Utils.lambda$fillMenu$6(Utils.java:349)
	at com.intellij.openapi.actionSystem.impl.Utils.computeWithRetries(Utils.java:898)
	at com.intellij.openapi.actionSystem.impl.Utils.fillMenu(Utils.java:348)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.fillMenu(ActionMenu.java:445)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.setPopupMenuVisible(ActionMenu.java:390)
	at java.desktop/javax.swing.JPopupMenu.menuSelectionChanged(JPopupMenu.java:1509)
	at java.desktop/javax.swing.MenuSelectionManager.setSelectedPath(MenuSelectionManager.java:134)
	at java.desktop/javax.swing.plaf.basic.BasicMenuUI$Handler.mouseEntered(BasicMenuUI.java:595)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6666)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3385)
	at com.intellij.openapi.actionSystem.impl.ActionMenu.processMouseEvent(ActionMenu.java:294)
	at java.desktop/java.awt.Component.processEvent(Component.java:6422)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5027)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4954)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEnterExit(Container.java:4732)
	at java.desktop/java.awt.LightweightDispatcher.trackMouseEnterExit(Container.java:4710)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4561)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4522)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2808)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4855)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:791)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:740)
	at java.desktop/java.awt.EventQueue$3.run(EventQueue.java:734)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:97)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:764)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:762)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:761)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:685)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:633)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:588)
	at com.intellij.ide.IdeEventQueue.access$_dispatchEvent(IdeEventQueue.kt:67)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:369)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1$1.compute(IdeEventQueue.kt:368)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:787)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:368)
	at com.intellij.ide.IdeEventQueue$dispatchEvent$processEventRunnable$1$1.invoke(IdeEventQueue.kt:363)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$1(IdeEventQueue.kt:992)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:113)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:992)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$7(IdeEventQueue.kt:363)
	at com.intellij.openapi.application.impl.ApplicationImpl.runIntendedWriteActionOnCurrentThread(ApplicationImpl.java:861)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:405)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)

```

**Comment by MCMicS** [comment]
create Intellij issue; https://youtrack.jetbrains.com/issue/QD-6397/java.lang.IllegalArgumentException-miter-limit-1

**Comment by weisJ** [comment]
Fixed in upstream jsvg: https://github.com/weisJ/jsvg/commit/ebb6773734cf105715e93d40f7da8d4e2afc069a

**Comment by MCMicS** [comment]
released with [0.13.18](../releases/tag/v0.13.18)
